community/yarn/APKBUILD

```diff
@@ -10,34 +10,14 @@ license="BSD-2"
 depends="nodejs"
 options="!check"
 source="https://yarnpkg.com/downloads/$pkgver/$pkgname-v$pkgver.tar.gz
-	apk-install-method.patch
-	remove-support-for-legacy-nodejs.patch"
+	apk-install-method.patch"
 builddir="$srcdir/dist"
 
 prepare() {
 	default_prepare || return 1
 
-	cd "$builddir"
-
-	# Remove support for legacy nodejs.
-	rm -r lib-legacy
-
-	# Remove some unneeded files.
-	# Node does not load *.min.js files, so these are useless for us.
-	# We respect licenses, but really don't wanna waste disk space (~1 MiB)
-	# with dozens copies of the same files.
-	rm yarn.lock jenkins_jobs.groovy
-	find . \( -iname '*.md' \
-		-o -iname '*.md~' \
-		-o -iname '*.cmd' \
-		-o -name '*.min.js' \
-		-o -name '*.map' \
-		-o -name 'bower.json' \
-		-o -name 'Dockerfile.*' \
-		-o -name 'Gruntfile.js' \
-		-o -iname 'license*' -a ! -name '*.js' \
-		-o -type f -a -name '.*' \) \
-		-delete
```

Comment:
Yes the entire `node_modules` folder is gone, there are just 9 files in the new tarball, and we remove the 2 `.cmd` files leaving 7...

I don't think we should remove the `LICENSE` file, we did it before because there were many copies of the same file in `node_modules`, this is just one file, we should respect the licence by including it.